src/components/InstallButton.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Dropdown, Link } from '@newrelic/gatsby-theme-newrelic';
+import { css } from '@emotion/react';
+
+const sampleItems = new Array(10).fill().map((_, i) => i + 1);
+
+const createMenuItems = (items) => items.map((i) => <Dropdown.MenuItem>{i}</Dropdown.MenuItem>)
+  const menuItems = [];
```

Comment:
Looks like those auto changes screwed up a bit here 🙃

Reply:
whoopsy

Reply:
i'll take another look at this not sure what's going on

Reply:
🤷

Reply:
aha ok it kept part of both functions for some reason, should be fixed now 😀